openslides_backend/models/__init__.py

```diff
@@ -0,0 +1 @@
+from .assert_migration_index import assert_migration_index  # noqa
```

Comment:
In the backend, we do not use the `__init__.py` for such exports, so I don't think it's fitting to use it here in this way